From 5ae8dc30c2d7dae7a50f8455c510b8ad9cb51f52 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 15 May 2020 21:47:43 +0300
Subject: [PATCH] [SW-11875] goya2: ppboot: use arcboot export elf

Since ppboot will use arcboot symbols to verify preboot image, update
makefile to use link exports.elf file for ppboot image compilation.

This change applies only to goya2_arc target. Also assumption is made
that exports.elf will be available when complete goya2_arc target is
compiled

Change-Id: I563fce849b0de94173986d7317efbf4efa75db80
---

diff --git a/Makefile b/Makefile
index d525910..2c0bc22 100644
--- a/Makefile
+++ b/Makefile
@@ -53,6 +53,7 @@
 # compile binaries based on target
 ifeq ($(HL_DEV),goya2_arc)
 	BINARIES = $(PPBOOT_TARGET) $(PCI_PHY_FW_TARGET)
+	LDFLAGS += -Wl,--just-symbols ../arcboot-custom/exports.elf
 else ifeq ($(HL_DEV),goya2_arc_gnu)
 	BINARIES = $(PREBOOT_TARGET)
 else
